.circleci/config.yml

```diff
@@ -117,9 +138,8 @@ jobs:
                 name: Prepare Slack message
                 when: always
                 command: |
-                  # Get the name of the device that is running. Using "<< parameters.device >>" can cause slack formatting errors.
```

Comment:
> Get the name of the device that is running. Using "<< parameters.device >>" can cause slack formatting errors.

I've changed this to use `'<< parameters.device >>'` to see if that still works, otherwise we'd have to keep this `xcrun simctl list` logic — which might not find the same device that Fastlane used for tests.